### PR TITLE
Adding access level parameter for pipeline create REST API info

### PR DESCRIPTION
**Pull request by karensawrey**
This PR adds the info about `teams` parameter that replaces the deprecated `team_uuids` parameter.

### CODE REVIEW

**Review by olyism — CHANGES_REQUESTED** [review]

**Comment by karensawrey** [comment]
Thank you so much for the help, @olyism!
Now I need to get rid of the trailing " in there, but it already looks the way it's supposed to.

File: pages/apis/rest_api/pipelines.md.erb [hunk 1]
```diff
@@ -330,8 +330,12 @@ Optional [request body properties](/docs/api#request-body-properties):
...
   </tr>
   <tr>
-    <th><code>team_uuids</code></th>
-    <td>An array of team UUIDs to add this pipeline to. You can find your team's UUID either using the <a href="/docs/apis/graphql-api">GraphQL API</a>, or on the settings page for a team. This property is only available if your organization has enabled Teams.<p class="Docs__api-param-eg"><em>Example:</em> <code>["14e9501c-69fe-4cda-ae07-daea9ca3afd3"]</code></p></td>
+    <th><code>teams</code></th>
+    <td>An array of team UUIDs to add this pipeline to. Allows you to very access level for the pipeline in a team. The available access level options are <code>read_only</code>, <code>build_and_read</code>, and <code>manage_build_and_read</code> You can find your team's UUID either using the <a href="/docs/apis/graphql-api">GraphQL API</a>, or on the settings page for a team. This property is only available if your organization has enabled Teams. Replaces deprecated <code>team_uuids</code> parameter.<p class="Docs__api-param-eg"><em>Example:</em>
```
Comment (pzeballos) [inline pages/apis/rest_api/pipelines.md.erb:334]:
```suggestion
    <td>An array of team UUIDs to add this pipeline to. Allows you to specify the access level for the pipeline in a team. The available access level options are <code>read_only</code>, <code>build_and_read</code>, and <code>manage_build_and_read</code> You can find your team's UUID either using the <a href="/docs/apis/graphql-api">GraphQL API</a>, or on the settings page for a team. This property is only available if your organization has enabled Teams. Replaces deprecated <code>team_uuids</code> parameter.<p class="Docs__api-param-eg"><em>Example:</em>
```

Comment (pzeballos) [reply pages/apis/rest_api/pipelines.md.erb:334]: Not sure I get the `very` on this line, I updated a bit the text

File: pages/apis/rest_api/pipelines.md.erb [hunk 2]
```diff
@@ -577,8 +581,12 @@ Optional [request body properties](/docs/api#request-body-properties):
...
```
Comment (pzeballos) [inline pages/apis/rest_api/pipelines.md.erb:585]:
```suggestion
    <td>An array of team UUIDs to add this pipeline to. Allows you to specify the access level for the pipeline in a team. The available access level options are <code>read_only</code>, <code>build_and_read</code>, and <code>manage_build_and_read</code> You can find your team's UUID either using the <a href="/docs/apis/graphql-api">GraphQL API</a>, or on the settings page for a team. This property is only available if your organization has enabled Teams. Replaces deprecated <code>team_uuids</code> parameter.<p class="Docs__api-param-eg"><em>Example:</em>
```

Comment (pzeballos) [reply pages/apis/rest_api/pipelines.md.erb:585]: Same here

**Review by pzeballos — CHANGES_REQUESTED** [review]
It's not clear to me a sentence, but the rest looks good 👍🏻

**Comment by karensawrey** [comment]
@pzeballos yeah, that "very" was supposed to be "specify" - some rogue autocorrect at work, sorry. Thanks for catching it!

**Review by pzeballos — APPROVED** [review]

**Comment by karensawrey** [comment]
Let's get this one rollin`.

**Pull request merged**